bin/filter/filter_service.py

```diff
@@ -21,15 +27,48 @@ def main():
 
     reader = csv.DictReader(sys.stdin)
     outnames = ['owner_org'] + [f for f in reader.fieldnames
-        if f not in REMOVE_COLUMNS and f != 'owner_org']
+        if f not in REMOVE_COLUMNS and f != 'owner_org'] \
+               + ['num_applications_total'] + ['program_name_en'] + ['program_name_fr']
     writer = csv.DictWriter(sys.stdout, outnames)
     writer.writeheader()
     for row in reader:
-        try:
-            for rem in REMOVE_COLUMNS:
-                del row[rem]
-            writer.writerow(row)
-        except ValueError:
-            pass
+        for rem in REMOVE_COLUMNS:
+            del row[rem]
+
+        # calculate total number of applications
+        row['num_applications_total'] = 0
+        num_fields = [ 'num_phone_enquiries',
+                       'num_applications_by_phone',
+                       'num_website_visits',
```

Comment:
Same as `num_phone_enquiries`, I don't think we need `num_website_visits` as we are just calculating the `num_applications_total`